std_msgs/Header header  # frame_id and time stamp
shape_msgs/Mesh mesh    # triangular mesh composed of triangles and vertices
float64[]       u       # horizontal texture coordinates for each mesh vertex
float64[]       v       # vertical texture coordinates for each mesh vertex
